ruby/ql/test/library-tests/dataflow/type-tracker/TypeTracker.expected

```diff
@@ -43,10 +43,6 @@ track
 | type_tracker.rb:12:1:16:3 | self in m | type tracker with call steps | type_tracker.rb:12:1:16:3 | self (m) |
 | type_tracker.rb:12:1:16:3 | self in m | type tracker without call steps | type_tracker.rb:12:1:16:3 | self in m |
 | type_tracker.rb:13:5:13:7 | var | type tracker without call steps | type_tracker.rb:13:5:13:7 | var |
-| type_tracker.rb:13:5:13:23 | ... = ... | type tracker with call steps | type_tracker.rb:2:5:5:7 | self (field=) |
```

Comment:
For reference: The reason that `... - ...` is (correctly) no longer a `LocalSourceNode` is because it represents the `SsaDefinitionNode` for the assignment, not the `ExprNode`.